### ISSUE TITLE
Login by PIN codes & QR codes

### ISSUE THREAD

**Issue by sklirg** [dotkom/Nibble2#51]
Instead of typing in username/password of OW4 user account (which might be very complicated if you have a complex password and are used to a password manager), we should look into simple ways to log a person in.

This can be done by showing a unique code to the user logging in, and make them log in and authenticate that login from another device (like how [Youtube TV](https://www.youtube.com/tv) makes you authenticate on a TV). This could further be simplified by generating a URL that will authorize the login and show this as a QR code which the user can scan with their phone and thereby log in.

This would also reduce the need for an on-screen keyboard.

Note: This would also have to be implemented in [dotkom/onlineweb4](https://github.com/dotkom/onlineweb4).

**Comment by Aborysa** [comment]
Was implemented in #57

**Comment by sklirg** [comment]
Not really, that feature simplified the registration process, not the log
in process.

**Comment by Aborysa** [comment]
Oops, my bad.
